### ISSUE TITLE
Font unreadable on xfce4-panel applets

### ISSUE THREAD

**Issue by gruentee** [oberon2007/vertex-maia-themes#3]
When hovering over the applets, the letters become readable again.
Affected applets: Date/Time applet, power manager applet

See attached screenshot.

![xfce4-panel_020](https://cloud.githubusercontent.com/assets/923759/20593822/0cb8d168-b234-11e6-9e7e-7ff5cb3b1065.png)

**Comment by oberon-manjaro** [comment]
Hm. I don't know what exactly can be different with your settings, but I don't see that issue:
![](http://imgur.com/RLb3TGJ.png)

**Comment by gruentee** [comment]
@oberon2007: 
I tried it on my work computer (apparently same config) and it works fine. Will have to check again.

**Comment by oberon-manjaro** [comment]
Try to reinstall the their package! Maybe your panel.rc file got screwed somehow...? :wink:

**Comment by gruentee** [comment]
Where is this file located?

I tried re-installing, deleting `~/.config/xfce4-panel` folder, killing/restarting the panel – none of it helped.

**Comment by oberon-manjaro** [comment]
The file lives in **/usr/share/themes/Vertex-Maia/gtk-2.0/panel.rc** and will be rewritten when you reinstall the theme. Maybe a stupid question but, uhm ... are you sure you have set the correct theme? :wink: Have you tried switching themes anyway? What happens then?

**Comment by gruentee** [comment]
OK, I misunderstood: I tried reinstalling the panel, because I suspected said file to be part of its configuration. 
Re-installing vertex-maia-themes package doesn't help either. 

The dark versions of the theme are not affected: fonts display correctly.

**Comment by oberon-manjaro** [comment]
https://forum.manjaro.org/t/xfce4-power-manager-1-6-badly-visible-on-panel/11537/12
I don't think this is an issue of the theme actually. Seems to be related to the panel and the plugins themselves ...

**Comment by Photon89** [comment]
I got a reply concerning this issue: https://bugzilla.xfce.org/show_bug.cgi?id=13355 Surprisingly it turned out that it is necessary to fix it within the theme by adding

` #xfce4-power-manager-plugin { color: #ededed; }`

to `gtk-3.0/gtk.css`. Tested it, works for me. Don't know if the time applet still shows this problem (at least I cannot reproduce it) but probably it would need a similar line.

**Comment by gruentee** [comment]
@Photon89 Thanks for the info.

I just tested again and **xfce-power-manager-plugin** shows the correct font color now (without any stylesheet changes), **xfce4-datetime-plugin** not though.

**Comment by Photon89** [comment]
Right, I can confirm that, I didn't have the right plugin installed previously. For some reason a similar line as above 

`#xfce4-datetime-plugin { color: #ededed; }`

does not fix the problem...

**Comment by oberon-manjaro** [comment]
it this solved?

**Comment by Photon89** [comment]
Not yet. You can test it by installing the xfce4-datetime-plugin package (it is not included in the standard install).

**Comment by oberon-manjaro** [comment]
I don't have an active xfce install and cannot test this - in principle I'm afraid it's gonna be hard to provide gtk3 properties for every xfce4-plugin out there. If someone wants to research the details, no problem... 😉!

**Comment by Photon89** [comment]
Another gtk3 panel plugin, xfce4-pulseaudio-plugin, has wrong colors: 
![pulse-icon](https://user-images.githubusercontent.com/1191480/31738715-db4486e8-b44b-11e7-9de7-1e81c03f2561.png)

edit: This seems to have got fixed at some point, at least right now end of December 2017) the icon is fine!

**Comment by Photon89** [comment]
Seems to be solved now as well?

**Comment by oberon-manjaro** [comment]
The issue with text is solved. No idea yet what can be done about the icons. It works with some icon-themes. Problem with others ...

**Comment by Photon89** [comment]
Hmm, which icon themes and plugins are affected by this?
